Machine Learning/sklearn examples/Support Vector Machines(SVM)/SVM.ipynb

```diff
@@ -0,0 +1,1041 @@
+{
```

Comment:
<p>Please use this code to create the buttons, I am well aware that these links will only work after the merge:</p><p> </p><pre class="ql-syntax" spellcheck="false">&lt;table class="tfo-notebook-buttons" align="left"&gt;
  &lt;td&gt;
    &lt;a target="_blank" href="https://colab.research.google.com/github/Rishit-dagli/Python3-by-practice/blob/master/Machine%20Learning/sklearn%20examples/Support%20Vector%20Machines(SVM)/SVM.ipynb"&gt;&lt;img src="https://www.tensorflow.org/images/colab_logo_32px.png" /&gt;Run in Google Colab&lt;/a&gt;
  &lt;/td&gt;
  &lt;td&gt;
    &lt;a target="_blank" href="https://github.com/Rishit-dagli/Python3-by-practice/blob/master/Machine%20Learning/sklearn%20examples/Support%20Vector%20Machines(SVM)/SVM.ipynb"&gt;&lt;img src="https://www.tensorflow.org/images/GitHub-Mark-32px.png" /&gt;View source on GitHub&lt;/a&gt;
  &lt;/td&gt;
&lt;/table&gt;
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Rishit-dagli/Python3-by-practice/pull/81/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>